### PR TITLE
pangram: Add case-insensitive test

### PR DESCRIPTION
**Pull request by behrtam**
see #266

### CODE REVIEW

File: exercises/pangram/canonical-data.json [hunk 1]
```diff
@@ -49,6 +49,11 @@
          "description": "Panagram in alphabet other than ASCII",
          "input": "Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.",
          "expected": false
+      },
+      {
+        "description": "test that count is case-insensitive",
```
Comment (Insti) [inline exercises/pangram/canonical-data.json]: This needs a better description of what we're testing for.
case insensitivity is already covered by: "Five quacking Zephyrs jolt my wax bed."

Comment (Insti) [inline exercises/pangram/canonical-data.json]: How about: "upper and lower case versions of the same character should not be counted separately"

**Comment by kytrinyx** [comment]
@behrtam Here also a more descriptive commit message would be really helpful.

**Comment by behrtam** [comment]
> I think there's a better description than that, I just don't know what it is right now.

Any new ideas or someone else?

**Comment by Insti** [comment]
No new ideas, "upper and lower case versions of the same character should not be counted separately" is the best I have for now.

**Comment by stkent** [comment]
+1 to "upper and lower case versions of the same character should not be counted separately"; it's a little verbose, but perfectly clear.

**Comment by behrtam** [comment]
Perfect, thanks both of you. Than I guess we are good to go.

**Comment by Insti** [comment]
Thanks @behrtam :heart:

**Pull request merged**